packages/ksf-login/src/KSF/Login/Component.purs

```diff
@@ -101,13 +103,16 @@ data Action =
   | FormPassword String
   | MergeCancel
   | SetMergeInfo (Maybe MergeInfo)
+  | SetViewStep View.LoginViewStep
```

Comment:
This is to set "steps" in the login/registration/user flow. I would have initially wanted to separate the account merging stuff from the login component and use it as a "step" as well.